rest/route/host.go

```diff
@@ -449,6 +449,11 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 
 	if !ch.dryRun {
+		err = ch.sc.RemoveAdminFromProjects(ch.user)
+		if err != nil {
```

Comment:
nit: you can put 452 and 453 on the same line with ` if err = ch.sc.RemoveAdminFromProjects(ch.user); err != nil`
We frequently do this when the function call is short, and when the only return value is an error